js/menu.js

```diff
@@ -7,21 +9,31 @@ export default class Menu {
     this.navbarId = navbarSelector.replace(".", "#");
     this.menus = this.navbar ? this.navbar.querySelectorAll("details") : null;
     this.navOverlay = document.querySelector(".nav-overlay");
+    this.mainMenu = document.getElementById("navbar");
+    this.mainMenuItems = this.mainMenu ? this.mainMenu.querySelectorAll("details") : null;
```

Comment:
This is sort of a wonky approach I'm not keen on, because it's going to run on every instantiation of the object, and consequentially affect something that potentially isn't part of it. For instance, This is still going to run when the filter bar is instantiated, which will result in a duplicate set of onToggle handlers binding to the main nav, even though the code is related to the filter bar.

Let's take a more object oriented approach to this. On line 7, consider making the following change:
`constructor(navbarSelector, generateOverlay = false) {`

Then, ditch lines 12 and 13 entirely, we should be able to rely on line 10 for the same thing. And instead of using line 11 and hunting out a div in the HTML, let the JS create the overlay on demand, like:
`this.navOverlay = document.createElement('div');
this.navOverlay.classList.add('nav-overlay');`

Later down on 65, you just check if generateOverlay is true, if it is, appendChild this.navOverlay to the body, and removeChild when it goes away.

This way, the existence and behavior of the overlay is entirely dependent on the settings of the object, and we aren't forcing behavior to leak out of the object to elements that might not be relevant to it (e.g. filter-bar)